[Qt] Reintroduce networkstyle to title texts

Fuzzbawls opened the pull request:
Allows for the window's title to contain `[testnet]` or `[regtest]` as
needed.

Also makes the splash screen a full window.

window with 96b4e09:
![Screen Shot 2019-09-15 at 12 23 57 AM](https://user-images.githubusercontent.com/7393257/64928842-6e96ee80-d7d2-11e9-9600-682a89390de6.png)

window with 4bfb2f6:
![Screen Shot 2019-09-15 at 1 41 12 AM](https://user-images.githubusercontent.com/7393257/64928849-7c4c7400-d7d2-11e9-811f-750b72ac42ae.png)

furszy commented:
It was the intended behavior to be a frameless window. Look at the borders, rounded corners and the image is rounded prepared for that too (Most visible is at the top right and top left corner in which the current window shows a white background).

Would prefer to leave it like before and if we want to add dialog movement in the future, add the functionality directly to the view without add the OS frame. It doesn't looks good.

furszy commented:
Updating the PR following the meeting convo, all good from my side. 
Basically the discussion was about the ugliness of the OS default window frame and the required time to do it nicely with all of the standard functionality.
So, we move now with the splash dialog with the OS default frame now and implement a good custom designed top bar when have some extra time, not a priority atm.

Fuzzbawls commented:
I've added two additional commits here:
- 96b4e09 resolves the "pinstripe" effect with gitian macOS builds
- 4bfb2f6 allows for programatic "painting" text on top of the splash image (used to display the version here)

I've added screenshots of how it looks to the OP
